lints/lint_qcstatem_qclimitvalue_valid.go

```diff
@@ -16,10 +16,9 @@ package lints
 
 import (
 	"encoding/asn1"
-	"unicode"
-
 	"github.com/zmap/zcrypto/x509"
 	"github.com/zmap/zlint/util"
+	"unicode"
```

Comment:
This import shouldn't have been rearranged.

Reply:
I think my editor's `goimports` version must differ from the author's. Fixed in 6fdf81d.

Thanks for the :mag: @titanous!